network/api/v1/api.go

```diff
@@ -89,3 +90,15 @@ func (a Wrapper) GetTransactionPayload(ctx echo.Context, hashAsString string) er
 	_, err = ctx.Response().Writer.Write(data)
 	return err
 }
+
+// RenderGraph visualizes the DAG as Graphviz/dot graph
+func (a Wrapper) RenderGraph(ctx echo.Context) error {
+	visitor := dag.NewDotGraphVisitor(dag.ShowShortRefLabelStyle)
+	err := a.Service.Walk(visitor.Accept)
+	if err != nil {
+		log.Logger().Errorf("Error while rendering graph: %v", err)
+		return ctx.String(http.StatusInternalServerError, err.Error())
```

Comment:
`core.NewProblem` is the new way of returning errors.

Reply:
I'll do that in one go (no pun intended) for all of `network`, instead of bit by bit